SWLOR.Game.Server/Feature/GuiDefinition/ViewModel/ShipManagementViewModel.cs

```diff
@@ -923,6 +945,23 @@ private void LoadShip()
                     LowPower8Tooltip = string.Empty;
                 }
 
+                ConfigurationVisible = shipDetail.ConfigurationNodes >= 1;
+
+                module = ship.Status.ConfigurationModules.ContainsKey(1)
```

Comment:
I would do a count check here. If we add a second or third configuration module in the future, this will break if the user puts a module in those slots instead of the first one.

Reply:
I've implemented the code for up to 8 config slots. Maybe with corvettes when I add them I'll allow for up to two configuration slots to be used on them. It would open up a lot of customization options I think.

Reply:
This isn't doing what I thought it was, disregard.